lte/gateway/python/integ_tests/s1aptests/test_attach_ics_drop_with_mme_restart.py

```diff
@@ -129,7 +129,11 @@ def test_attach_ics_drop_with_mme_restart(self):
             s1ap_types.tfwCmd.UE_SET_DROP_ICS, drop_init_ctxt_setup_req,
         )
 
-        self._s1ap_wrapper.magmad_util.wait_for_restart_to_finish(30)
+        wait_for_restart = 30
+        self._s1ap_wrapper.magmad_util.wait_for_restart_to_finish(
+            ["mme", "mobilityd", "pipelined", "sessiond"],
+            wait_for_restart
+        )
 
```

Comment:
Why put the restart check here, instead of adding the 30s maximum wait in L119 where `mme` is actually restarted?

Reply:
The waiting time was at this point before, but I see how that might not make much sense after the restructuring of the restart (and possibly neither before that)